link_linux.go

```diff
@@ -13,6 +13,14 @@ import (
 var native = nl.NativeEndian()
 var lookupByDump = false
 
+var macvlanModes = [...]uint32{
```

Comment:
I'm just following the convention where the constants from Linux headers are in `nl` package. We also need to ensure that 0 value maps to a noop. That's technically possible in this case as the constants start with 1 (for some reason they look like they can be combined in a bitmask), however it seemed a bit magical to do that. But I'm open to getting rid of double constants.